tests/core/component.test.js

```diff
@@ -27,7 +27,7 @@ suite('Component', function () {
     });
   });
 
-  suite('buildData', function () {
+  suite('recomputeData', function () {
```

Comment:
The outcome of `recomputeData` is roughly the same as `buildData`. One difference being that `recomputeData` doesn't return the updated data value, requiring these test cases to be changed accordingly.